### PR TITLE
Adding updates around open source calls

### PR DESCRIPTION
**Pull request by kris-nova**
Defining office hours vs repo planning and removing broken mailing list links

Signed-off-by: Kris Nova <kris@nivenly.com>

**What type of PR is this?**

/kind documentation

**Any specific area of the project related to this PR?**

NONE

**What this PR does / why we need it**:

Removing confusion from the README.

**Which issue(s) this PR fixes**:

Fixes https://github.com/falcosecurity/office-hours/issues/31

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Updating documentation for office ours and repo planning calls
```

### CODE REVIEW

**Comment by kris-nova** [comment]
closes https://github.com/falcosecurity/office-hours/issues/31

File: README.md [hunk 1]
```diff
@@ -45,14 +45,28 @@ See [Falco Documentation](https://falco.org/docs/) to quickly get started using
...
 ---
 
-Falco has bi-weekly office hour style meetings where we plan our work on the project. You can get a Google calendar invite by joining the mailing list. It will automatically be sent.
+Falco has bi-weekly "office hours" which is an open call to discuss Falco from a user perspective. These happen on opposite weeks of Repo planning calls. 
+
+> Are you using Falco? Do you have have ideas for things to do with Falco? How can Falco be better?
+
+Note: We are in the process of curating an open Falco mailing list and Calendar via the CNCF. Stay tuned for detail.
+
+Wednesdays at 8am Pacific on [Zoom](https://sysdig.zoom.us/j/213235330).
```
Comment (fntlnz) [inline README.md]: It's both calls on Wednesday?

File: README.md [hunk 2]
```diff
@@ -45,14 +45,28 @@ See [Falco Documentation](https://falco.org/docs/) to quickly get started using
...
 ---
-* [Join the mailing list](http://bit.ly/2Mu0wXA) for news and a Google calendar invite for our Falco open source meetings. Note: this is the only way to get a calendar invite for our open meetings.
 * [Website](https://falco.org) for Falco.
 * Join our [Public Slack](https://slack.sysdig.com) channel for open source Sysdig and Falco announcements and discussions.
 
 Office hours
 ---
 
-Falco has bi-weekly office hour style meetings where we plan our work on the project. You can get a Google calendar invite by joining the mailing list. It will automatically be sent.
+Falco has bi-weekly "office hours" which is an open call to discuss Falco from a user perspective. These happen on opposite weeks of Repo planning calls.
```
Comment (leodido) [inline README.md]:
```suggestion
Falco has bi-weekly [office hours](https://github.com/falcosecurity/office-hours) which is an open call to discuss Falco from a user perspective. These happen on opposite weeks of Repo planning calls. 
```

File: README.md [hunk 3]
```diff
@@ -45,14 +45,28 @@ See [Falco Documentation](https://falco.org/docs/) to quickly get started using
...
 
 Office hours
 ---
 
-Falco has bi-weekly office hour style meetings where we plan our work on the project. You can get a Google calendar invite by joining the mailing list. It will automatically be sent.
+Falco has bi-weekly "office hours" which is an open call to discuss Falco from a user perspective. These happen on opposite weeks of Repo planning calls. 
+
+> Are you using Falco? Do you have have ideas for things to do with Falco? How can Falco be better?
+
+Note: We are in the process of curating an open Falco mailing list and Calendar via the CNCF. Stay tuned for detail.
```
Comment (leodido) [inline README.md]:
```suggestion
```

File: README.md (hunk 1 above)
Comment (leodido) [inline README.md]:
```suggestion
[Wednesdays at 8am Pacific](https://lists.cncf.io/g/cncf-falco-dev/calendar) on [Zoom](https://sysdig.zoom.us/j/213235330).
```

File: README.md [hunk 4]
```diff
@@ -45,14 +45,28 @@ See [Falco Documentation](https://falco.org/docs/) to quickly get started using
...
+Repo planning 
+--- 
+
+Falco has bi-weekly planning meetings which is an open call to discuss upcoming Falco releases, and assign open GitHub issues to engineers. These happen on opposite weeks of office hours calls. 
+
+> Do you want to contribute to Falco? Are you interested in working on Falco? Do you want to fix something or make something better?
+
+Note: We are in the process of curating an open Falco mailing list and Calendar via the CNCF. Stay tuned for detail.
 
 Wednesdays at 8am Pacific on [Zoom](https://sysdig.zoom.us/j/213235330).
```
Comment (leodido) [inline README.md]:
```suggestion
[Wednesdays at 8am Pacific](https://lists.cncf.io/g/cncf-falco-dev/calendar) on [Zoom](https://sysdig.zoom.us/j/213235330).
```

**Comment by leodido** [comment]
@kris-nova I updated this PR to reflect the decisions we made (eg., adding the CNCF calendar link etc.)

Woudl you mind picking-up the suggestions here from the GitHub UI please? :) 
I do not have access to your fork thus I cannot add them myself.

**Comment by leodido** [comment]
@kris-nova would you mind picking-up the suggestions here from the GitHub UI please? :)

**Pull request closed without merging**